src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,28 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double fuelTankCap;
+    private int boomLength;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. It's best practice to keep them private and provide getters and setters for accessing these fields.